nvm.sh

```diff
@@ -695,6 +695,82 @@ nvm_binary_available() {
   nvm_version_greater_than_or_equal_to "$(nvm_strip_iojs_prefix "${1-}")" v0.8.6
 }
 
+nvm_set_colors() {
+  if [ nvm_has_colors ]
+  then
+    COLORS=$1
+    INSTALLED_COLOR="$(nvm_print_color_code ${COLORS:0:1})"
+    LTS_AND_SYSTEM_COLOR="$(nvm_print_color_code ${COLORS:1:1})"
+    CURRENT_COLOR="$(nvm_print_color_code ${COLORS:2:1})"
+    NOT_INSTALLED_COLOR="$(nvm_print_color_code ${COLORS:3:1})"
+    DEFAULT_COLOR="$(nvm_print_color_code ${COLORS:4:1})"
```

Comment:
each of these variables needs a `local` declaration on the line above it

Reply:
`local` declaration added